TRANSLATING.md

```diff
@@ -0,0 +1,51 @@
+Translation and Internationalization
+====================================
+
```

Comment:
If possible, it would be helpful to have an overview of the internationalisation infrastructure, how it works in relation to the site source file structure, and what needs to be done in future PRs when changing existing pages or adding a new page.

Reply:
I expanded this section a bit on the functionality of the plugin I used for this patch.

Reply:
> If possible, it would be helpful to have an overview of the internationalisation infrastructure ...

I would like to note that this description may also contain some specification or general criteria for the translation, for example, how to  translate words or expressions without direct equivalent between the default language (i.e, English) and to which it will be translated?

Reply:
Thanks @awvwgk, this has cleared up a lot of my questions 👍

Reply:
@aslozada I collected the links from the Discourse thread and added them as a resources section. We might as well just move those to the wiki at some point, because it is easier to view and edit.